pr_agent/servers/github_action_runner.py

```diff
@@ -50,7 +50,7 @@ async def run_action():
     # Handle pull request event
     if GITHUB_EVENT_NAME == "pull_request":
         action = event_payload.get("action")
-        if action in ["opened", "reopened"]:
+        if action in ["opened", "reopened", "synchronize", "ready_for_review"]:
```

Comment:
so the synchronize is debatable.
if one were to include/configure this trigger the AI bot would run on *every* PR push.

one could not include the trigger on the workflow level
or if we feel this is never intended/supported/unwanted we could remove it here to protect the operator